### DISCUSSION TITLE
Fraud website [fastpaydm.vercel.app]

### DISCUSSION THREAD

**Discussion by punyadya** [vercel/community#5164] [Account Issue] [answered]
### Question

Hi Team Vercel,

For information, as a staff of a company whose website is almost similar to the registered domain name on [vercel.com](http://vercel.com/), namely :

https://fastpaydm.vercel.app/

We checked that the website is not registered and was misused by irresponsible parties by taking user data, pins, keys when entering data on the website.
This has caused a lot of losses to our customers.

Our official website is:
https://loket2-fastpay.bmsecure.id/
and is very similar to the website that we informed before, our official website is our copyright and an offense because it has misused the same website by taking our customer data and causing losses for this.

For additional information, please close the website immediately:
--Username of the reported account or *.[vercel.app](http://vercel.app/) deployment URL: https://fastpaydm.vercel.app/
--Your company name: PT Bimasakti Multi Sinergi
--Company website: https://www.bm.co.id/
--Website Login Aplikasi: https://loket2-fastpay.bmsecure.id/
--Your trademarked word, symbol, etc.: FASTPAY
--Trademark registration number: IDM000404099
--Trademark registration office (e.g., USPTO); Kementerian Hukum & Hak Asasi Manusia Jl. HR. Rasuna Said Kav. 8-9, Jakarta Selatan, Jakarta, Indonesia
--Description of confusion: Website : A website is the same as a patented website and is misused for personal interests by taking our customer data as it is. (username, pin)
--Requested Action : Deletion of the infringing website or redirection of a trademarked user name to an existing company account
--Include the following statement : I would like to remove the site because it was not authorized by the trademark owner, or his agent, or the law.
--Also include the following statement: I swear, with the threat of punishment for false oaths, that the information in this notice is accurate and that I am the owner of the trademark, or authorized to act on behalf of the owner, on the exclusive rights allegedly infringed.

**Comment by swarnava** [comment, marked as answer]
If you believe that a Vercel deployment contains malicious or illegal content, please email [abuse@vercel.com](mailto:abuse@vercel.com). When doing this, please provide as much relevant information as possible, taking into consideration our [DMCA policy](https://vercel.com/legal/dmca-policy) and [trademark policy](https://vercel.com/legal/trademark-policy).

**Reply by punyadya** [reply]
We have emailed abuse@vercel.com but there has been no reply from vercel.

Please check immediately so that there are no victims.

![Screenshot 2023-12-12 105457](https://github.com/vercel/community/assets/44915603/535b2804-ab26-49e5-ace6-8e9c00d0d7ad)

**Comment by punyadya** [comment]
We have emailed [abuse@vercel.com](mailto:abuse@vercel.com) but there has been no reply from vercel.

Please check immediately so that there are no victims.

![Screenshot 2023-12-12 105457](https://github.com/vercel/community/assets/44915603/c0c28e4e-a3f7-4f3a-a19d-c080499adfb0)

**Reply by swarnava** [reply]
Our dedicated team will get back to you as soon as we have bandwith triaging previous request. You can keep only one report per deployment to get faster response.

**Comment by punyadya** [comment]
Hello Vercel team,

Has this case been traced and shut down for the website, because we checked until now the website can still be accessed and this is also very detrimental to us and our members who have been deceived by the website.

We emphasise again for you Vercel team so that you can immediately shut down the website because this has also violated the rules.
